asab/api/service.py

```diff
@@ -17,7 +17,7 @@
 
 asab.Config.add_defaults({
 	"general": {
-		"manifest": "",
+		"manifest": "app/MANIFEST.json",
```

Comment:
Is this indeed correct? It doesn't seem compatible with the default value handling that you do here: https://github.com/TeskaLabs/asab/blob/Refactor/Use-Kazoo-asab-zookeeper/asab/api/service.py#L34

Reply:
"" should trigger all the "magic" 

`"manifest": ""` is correct